e2e/images/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:16.04
```

Comment:
Do we need ubuntu or can we live with a smaller base like alpine? If not, we could use ubuntu-slim. https://github.com/kubernetes/ingress/tree/master/images/ubuntu-slim